inductiva/_cli/utils.py

```diff
@@ -11,6 +12,35 @@
 END_TAG = "# >>> INDUCTIVA END:"
 
 
+def positive_float(value):
+    try:
+        value = float(value)
+    except TypeError as _:
+        raise argparse.ArgumentTypeError(f"{value} is an invalid type.")
+    if value <= 0:
+        raise argparse.ArgumentTypeError(f"{value} must be a positive number.")
+    return value
+
+
+def add_watch_argument(subparser):
+    """Add watch flag to the subparser that prompts the commands every Nsecs."""
+    subparser.set_defaults(watchable=True)
+    subparser.add_argument("-w",
+                           "--watch",
+                           nargs="?",
+                           const=2.0,
+                           type=positive_float,
+                           help="Prompt the command every N seconds.")
+
+
+def remove_flags(string, flags):
```

Comment:
this function has a bug and will not work all the time:

```python
>>> remove_flags("inductiva hi -w -t 10", ["w", "watch"])
'inductiva hi 10'

>>> remove_flags("inductiva hi -w -t", ["w", "watch"])
'inductiva hi'
```

Reply:
Found a hacky way to fix the bug, now some extra spaces may still appear.